workflow-templates/dotnet-nuget-ci-cd.yml

```diff
@@ -0,0 +1,19 @@
+name: NuGet Package CI/CD
+
+on:
+  push:
+    branches:
+      - $default-branch
+  pull_request:
+    branches:
+      - $default-branch
+  workflow_dispatch:
+
+jobs:
+  nuget:
+    name: Build/Test/Publish
+    uses: vermeer-corp/it-core-actions/.github/workflows/dotnet-nuget-ci-cd.yml@v2
+    with:
+      dotnet-version: 6.0.x
```

Comment:
This yaml file doesn't have a upload-artifact parameter like the other one. Just wanted to make sure that it isn't needed here

Reply:
Good catch. Added `publish-package` as a boolean parameter.